.circleci/config.yml

```diff
@@ -5,18 +5,118 @@ references:
     client pricehistory bot nlp servers broker eventstore nginx
 
 commands:
+
+  # rewriting internal 'checkout' command to solve the issue with triggering forked PRs build with API
+  checkout:
+    steps:
+      - run:
+          name: Set up HOME environment variable
+          command: |
+            # Workaround old docker images with incorrect $HOME
+            # check https://github.com/docker/docker/issues/2968 for details
+            if [ "${HOME}" = "/" ]
+            then
+              export HOME=$(getent passwd $(id -un) | cut -d: -f6)
+            fi
+
+      - run:
+          name: Set up SSH access
+          command: |
+            mkdir -p ~/.ssh
+
+            echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
```

Comment:
seems that these are not our keys and are circle-ci keys:
https://discuss.circleci.com/t/with-workflows-cloning-another-repo-times-out/13470/5
https://discuss.circleci.com/t/shallow-clone-for-circleci-2-0-builds/20200